subprojects/packagefiles/cppzmq/meson.build

```
@@ -0,0 +1,62 @@
+project('cppzmq', 'cpp',
+  version : '4.8.1',
+  license : 'MIT',
+)
+
+libzmq_dep = dependency('libzmq')
+
+cppzmq_inc = include_directories('.')
+
+cppzmq_dep = declare_dependency(include_directories: cppzmq_inc, dependencies: libzmq_dep)
+
+# examples
+thread_dep = dependency('threads')
+
+ex_pubsub_multithread_inproc_exe = executable('pubsub_multithread_inproc',
+  'examples/pubsub_multithread_inproc.cpp',
```

I was wondering why I could not build this until I saw https://github.com/zeromq/cppzmq/pull/568

An absolutely filthy hack: 

```suggestion
  'examples/pubsub_multithread_inproc.cpp',
  # fixed upstream in https://github.com/zeromq/cppzmq/pull/568
  cpp_args: ['-include', 'thread'],
```

---

Ah thanks, that's easier than adding patch files.